.github/issuecomplete.yml

```diff
@@ -9,11 +9,7 @@ labelColor:
 
 # The text of the comment to add to the issue in addition to the label
 commentText: >
-  Hello! It doesn't seem like we have quite enough information to send this to 
-  a human yet to help out. We would love if you could provide more details about 
-  your issue by following the template without modifying any of the pre-filled
-  text. If you're looking for support, head over to our [Help Center](https://support.revenuecat.com/hc/en-us)
-  to get in touch with our team directly.
+  Hello! It doesn't seem like we have quite enough information to send this to a human yet to help out. We would love if you could provide more details about your issue by following the template without modifying any of the pre-filled text. If you're looking for support, head over to our [Community](https://community.revenuecat.com).
```

Comment:
I did like having this split into lines because it's much easier to review changes in gh, horizontal scrolling is a bit of a pain